### PR TITLE
ocamlcodoc.1

### PR DESCRIPTION
**Pull request by thierry-martinez**
Introducing a new package: ocamlcodoc

Many thanks!

### CODE REVIEW

File: packages/ocamlcodoc/ocamlcodoc.1/opam [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+synopsis: "Extract test code from doc-comments"
+description: "ocamlcodoc extracts the preformatted source code in OCaml documentation comments, i.e. the code delimited by {[ ... ]} in comments delimited by (** ... *). A typical usage is to write examples in documentation comments that can be extracted and tested."
+maintainer: "Thierry Martinez <Thierry.Martinez@inria.fr>"
+authors: "Thierry Martinez <Thierry.Martinez@inria.fr>"
+homepage: "https://gitlab.inria.fr/tmartine/ocamlcodoc"
+bug-reports: "https://gitlab.inria.fr/tmartine/ocamlcodoc/issues"
+license: "GPL-3"
+dev-repo: "git+https://gitlab.inria.fr/tmartine/ocamlcodoc.git"
+build: [["dune" "build"]]
+install: [
```
Comment (mseri) [inline packages/ocamlcodoc/ocamlcodoc.1/opam]: The install and remove stanza are not needed with dune.

File: packages/ocamlcodoc/ocamlcodoc.1/opam [hunk 2]
```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+synopsis: "Extract test code from doc-comments"
+description: "ocamlcodoc extracts the preformatted source code in OCaml documentation comments, i.e. the code delimited by {[ ... ]} in comments delimited by (** ... *). A typical usage is to write examples in documentation comments that can be extracted and tested."
+maintainer: "Thierry Martinez <Thierry.Martinez@inria.fr>"
+authors: "Thierry Martinez <Thierry.Martinez@inria.fr>"
+homepage: "https://gitlab.inria.fr/tmartine/ocamlcodoc"
+bug-reports: "https://gitlab.inria.fr/tmartine/ocamlcodoc/issues"
+license: "GPL-3"
+dev-repo: "git+https://gitlab.inria.fr/tmartine/ocamlcodoc.git"
+build: [["dune" "build"]]
```
Comment (mseri) [inline packages/ocamlcodoc/ocamlcodoc.1/opam]: The idiomatic call is
```suggestion
build: [["dune" "build" "-p" name "-j" jobs]]
```

**Comment by mseri** [comment]
Usually we use more digits in the version, e.g. `1.0.0`, unless you plan to do incremental versions as integer jumps I would recommend to add at least one zero there

**Comment by thierry-martinez** [comment]
@mseri Thanks for all your suggestions!

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**